sklearn/ensemble/_voting.py

```diff
@@ -637,6 +637,10 @@ class VotingRegressor(RegressorMixin, _BaseVoting):
     >>> er = er.fit(X, y)
     >>> len(er.estimators_)
     2
+
+    For a detailed example of utilizing
+    :class:`~sklearn.ensemble.VotingRegressor`
+    For a more detailed example, refer to :ref:sphx_glr_auto_examples_ensemble_plot_voting_regressor.py.
```

Comment:
```suggestion
    For a more detailed example, refer to
    :ref:`sphx_glr_auto_examples_ensemble_plot_voting_regressor.py`.
```

You left some redundant lines there, and the formatting and sphinx cross reference syntax is not correct.